### ISSUE TITLE
Add room info to attendance page meetings

### ISSUE THREAD

**Issue by Rahuldeb5** [stuysu/epsilon#251]
For each meeting on the attendance page, add the date as well as room information to make it more clear and understandable for the users about which meeting this was

**Comment by SnowyNate** [comment]
I believe the date is already present. Just the room information right?

**Comment by TheEgghead27** [comment]
Yup — you can add the room info next to (i.e. `12/13/2024, 3:35:00 PM — 351`) or below (i.e. two lines) if desired
This one is on me for forgetting to comment about 1a6c508a130c1976fcc23dba68e2162a30f51b76
